demo/package.json

```diff
@@ -6,7 +6,8 @@
     "build-storybook": "build-storybook",
     "storybook": "start-storybook -p 6006",
     "storybook:axe": "build-storybook && axe-storybook",
-    "storybook:axe-no-build": "axe-storybook"
+    "storybook:axe-no-build": "axe-storybook --build-dir 'storybook-static'",
```

Comment:
storybook-static is the default value for --build-dir. Did something change, and now we need to specify it here?

Reply:
Okay, I see. So the tests output to a specific place. And it seems this command runs in the test environment, so we need to specify a different build directory for this command.

We can leave this for now, and I'll try to think up some other options.